docs/src/main/asciidoc/datasource.adoc

```diff
@@ -815,6 +815,7 @@ When compiling a Quarkus application to a native image, the MySQL support for JM
 
 * The lack of JMX support is a natural consequence of running in native mode and is unlikely to be resolved.
 * The integration with OCI is not supported.
+* The OpenTelemetry integration included in the MySQL Connector/J driver is disabled in favor of the one provided by Agroal. If you really need to use the OpenTelemetry integration provided by the MySQL Connector/J driver, enable it explicitly by setting `quarkus.datasource.jdbc.additional-jdbc-properties.openTelemetry=PREFERRED`.
```

Comment:
We know if they activate it, it will fail with the extension. We should probably mention it.